modules/monix/src/weaver/monixcompat/suites.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package monixcompat
+
+import cats.effect.{ ConcurrentEffect, ContextShift, Resource, Timer }
+
+import monix.eval.Task
+import monix.eval.instances._
+import monix.execution.Scheduler
+
+trait BaseMonixSuite { self: ConcurrentEffectSuite[Task] =>
+  val ec                              = scala.concurrent.ExecutionContext.global
+  val scheduler                       = Scheduler(ec)
+  implicit def timer: Timer[Task]     = Task.timer(scheduler)
+  implicit def cs: ContextShift[Task] = Task.contextShift(scheduler)
+  implicit def effect: ConcurrentEffect[Task] =
+    new CatsConcurrentEffectForTask()(scheduler, Task.defaultOptions)
+}
+
+trait MutableMonixSuite extends MutableFSuite[Task] with BaseMonixSuite
+
+trait SimpleMutableMonixSuite extends MutableMonixSuite {
+  type Res = Unit
+  def sharedResource: Resource[Task, Unit] = Resource.pure(())
```

Comment:
Still looking into whether `Bracket` should be used instead of `Resource`. From my understanding, Monix `Task.bracket` simply creates a Cats Effect `Bracket`...

Reply:
Let's leave it as Resource. Monix doesn't have (afaik) a competing construct to `cats.effect.Resource`(zio however does). `Resource` makes use of the `Bracket` typeclass, which is implemented using `Task#bracket` in this case